### PR TITLE
feat: setting comment

### PR DESCRIPTION
**Pull request by mikelxk**
style: remove icon before fav playlist
chore: fix some eslint warning
style: sort imports

### CODE REVIEW

**Comment by listen1** [comment]
please fix fail checks before merge.

**Comment by mikelxk** [comment]
> please fix fail checks before merge.

I've managed to pass vue-tsc , but that's only a general workaround since the problem still exists.

File: src/components/Sidebar.vue [hunk 1]
```diff
@@ -70,34 +70,35 @@
           dragtype="application/listen1-favoriteplaylist"
           @click="$router.push(`/playlist/${i.id}`)"
           @drop="onSidebarPlaylistDrop('favorite', i.id, $event)">
-          <SidebarEntry :text="i.title" icon="vue-feather-disc"></SidebarEntry>
+          <Entry :text="i.title" />
```
Comment (listen1) [inline src/components/Sidebar.vue:73]: Please try to submit PR for one thing per time.

And I think remove icon before favorite songlist needs a good reason. Most of listen1 UI design follows music player of netease, qq, etc. So if there's some UI change that is different from these application, please add some comment for reason.

Comment (mikelxk) [reply src/components/Sidebar.vue:73]: Reverted. However, there are disk icons in front of every favorited list. I think we can add a disk icon in front of the text 'fav list' but adding the same icon on each entry cannot really provide any information to human and might waste text space.

Comment (listen1) [reply src/components/Sidebar.vue]: > Reverted. However, there are disk icons in front of every favorited list. I think we can add a disk icon in front of the text 'fav list' but adding the same icon on each entry cannot really provide any information to human and might waste text space.

yeah, that's a problem indeed. We can maybe make sidebar wider to leave more space for title. In my opinion, icon before title is entry style and so if my playlist has icon and fav playlist doesn't seems like something went wrong.

**Pull request merged**